_locales/_untranslated_en.json

```diff
@@ -56,6 +56,9 @@
   "show_webxdc_in_chat": {
     "message": "Show Private App in Chat"
   },
+  "receive_backup": {
+    "message": "Receiving backup from second device"
```

Comment:
Maybe some other wording is better? Idk.

Reply:
Dunno about other wording but "a second device" is correct.